core/src/main/java/org/stellar/anchor/event/models/Amount.java

```diff
@@ -0,0 +1,15 @@
+package org.stellar.anchor.event.models;
```

Comment:
Can we use https://github.com/stellar/java-stellar-anchor-sdk/blob/main/platform-apis/src/main/java/org/stellar/platform/apis/shared/Amount.java instead?

Reply:
will refactor in a separate PR